src/App.tsx

```diff
@@ -1,9 +1,166 @@
+              <Image width={20} height={20} src="src/assets/heart.svg" className="rounded" />
+              <Button className="offset-md-4" variant="outline-info text-dark">Show more</Button>
+              <Button className="offset-md-0" variant="info">Go somewhere</Button>
+            </Card.Body>
+
+          </Col>
+        </Row>
+        <p />
+        <Row>
+          <Col>
+            <Card.Img variant="top" src="src/assets/ny.png" />
+            <Card.Body>
+              <Card.Title className="text-secondary">I'll be home for Christmas</Card.Title>
+              <Card.Text className="text-dark">
+                by Vocal Ensemble Chapel Jam
+                <p className="text-success">December 27 at the Kirche</p>
+              </Card.Text>
+              <Image width={20} height={20} src="src/assets/heart.svg" className="rounded" />
+              <Button className="offset-md-4" variant="outline-info text-dark">Show more</Button>
+              <Button className="offset-md-0" variant="info">Go somewhere</Button>
+            </Card.Body>
+          </Col>
+          <Col>
+            <Card.Img variant="top" src="src/assets/oldny.png" />
+            <Card.Body>
+              <Card.Title className="text-secondary">Old New Year</Card.Title>
+              <Card.Text className="text-dark">
+                by Symphony Orchestra and Chamber Choir
+                <p className="text-success">January 15 at the Drama Theater</p>
+              </Card.Text>
+              <Image width={20} height={20} src="src/assets/heart.svg" className="rounded" />
+              <Button className="offset-md-4" variant="outline-info text-dark">Show more</Button>
+              <Button className="offset-md-0" variant="info">Go somewhere</Button>
+            </Card.Body>
+          </Col>
+          <Col>
+            <Card.Img variant="top" src="src/assets/musica.png" />
+            <Card.Body>
+              <Card.Title className="text-secondary">Advent Music</Card.Title>
+              <Card.Text className="text-dark">
+                by Symphony Orchestra
+                <p className="text-success">December 16 at the Kirche</p>
+              </Card.Text>
+              <Image width={20} height={20} src="src/assets/heart.svg" className="rounded" />
+              <Button className="offset-md-4" variant="outline-info text-dark">Show more</Button>
+              <Button className="offset-md-0" variant="info">Go somewhere</Button>
+            </Card.Body>
+          </Col>
+        </Row>
+        <p className="text-dark">_____________________________________________________________________________________________________________________________________________________________________</p>
+      </Container>
+    </main>
+    <footer className="py-5 my-5 bg-dark">
+      <Container className="px-4">
+        <p className="text-center text-white">Copyright &copy; INFOHAVE 2022</p>
+      </Container>
+    </footer>
+  </div>
```

Comment:
Instead of repeating the code create several component (https://reactjs.org/docs/components-and-props.html)

1. Header
2. Footer
3. Main (content wrapper)
4. Events (events wrapper, will decide how items should be displayd)
5. Event (event card)